component/commons/handler/pom.xml

```diff
@@ -0,0 +1,157 @@
+        <version>1.0.28-SNAPSHOT</version>
+        <relativePath>../../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>org.wso2.carbon.extension.identity.passwordexpiryvalidation.handler</artifactId>
+    <version>1.0.28-SNAPSHOT</version>
+    <packaging>bundle</packaging>
+    <name>WSO2 Carbon Extension - Password Expiry Validation - Handler</name>
+    <url>http://wso2.org</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.felix</groupId>
+            <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jacoco</groupId>
+            <artifactId>org.jacoco.agent</artifactId>
+            <classifier>runtime</classifier>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-testng-common</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-testng</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.event</artifactId>
+        </dependency>
+
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-scr-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>org.wso2.carbon.identity.password.expiry.validation.handler.internal
+                        </Private-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.identity.password.expiry.validation.handler.internal,
+                            org.wso2.carbon.identity.password.expiry.validation.handler.*
+                        </Export-Package>
+                        <Import-Package>
+                            org.wso2.carbon.identity.mgt.*; version="${carbon.identity.package.import.version.range}",
+                            org.apache.commons.logging.*; version="1.0.4",
```

Comment:
Lets add the version/version range to the properties section of the parent pom and then use it here?